proprietary/tokens/src/components/amsterdam/checkbox.tokens.json

```diff
@@ -0,0 +1,79 @@
+{
+  "amsterdam": {
+    "checkbox": {
+      "color": { "value": "{amsterdam.color.primary-black}" },
+      "checkmark": {
+        "box-shadow": {
```

Comment:
That would be a little confusing I think. `border-width` and `border-color` are existing css properties, we shouldn't use those names if we aren't actually setting those properties. 

> we’re setting the border color here, not a shadow

I disagree, we're setting the `box-shadow` here, it just happens to look like a border. Because 'border' means something in CSS-land, which shouldn't mix up those terms imo.